cmd/swapd/main.go

```diff
@@ -203,6 +203,11 @@ var (
 				Name:  flagUseExternalSigner,
 				Usage: "Use external signer, for usage with the swap UI",
 			},
+			&cli.BoolFlag{
+				Name:  flagClaimRelayer,
+				Usage: "Relay claims for other XMR makers and earn a modest fee per relayed transaction",
```

Comment:
```suggestion
				Usage: "Relay transactions for other XMR makers and earn a fee (0.009 ETH) per relayed transaction",
```

if there is no way for the relayer to set their fee, I'd just put it in the description

Reply:
Yeah, I didn't add a way for the relayer to customize their fee in this PR. It would be follow-up work if we decided that we need it. I'll use `Sprintf` when using your suggestion above so any changes to the relayer fee can be in one place.